addons/website_portal_sale/controllers/main.py

```diff
@@ -132,7 +130,7 @@ def portal_my_orders(self, page=1, date_begin=None, date_end=None, sortby=None,
 
         values.update({
             'date': date_begin,
-            'orders': orders,
+            'orders': orders.sudo(),
```

Comment:
To display the total, it required to have read access to product.pricelist, account.tax, ...
AL does not want portal to have access to pricelist.
Also, on the frontend form view, portal needs to list the product.product name of its SO lines, and craque dedans ! Donc sudo is requis ;)